test/e2e/serial/tests/tolerations.go

```diff
@@ -108,6 +108,7 @@ var _ = Describe("[serial][disruptive][rtetols] numaresources RTE tolerations su
 		})
 
 		When("[tier2] invalid tolerations are submitted ", Label("tier2"), func() {
+			const unsupportedValue = "Unsupported value"
```

Comment:
let's make it a top level constant or even better let's reuse constants from the library packages we consume, if possible